### PR TITLE
Feature structure

### PR DESCRIPTION
**Pull request by kimothokamau**
- This pull request has the create-react-app initial configuration
- Linters config
- Folder structure

### CODE REVIEW

**Review by Dipeshtwis — COMMENTED** [review]
Hi @kimothokamau,

# Status: Invalid ❌

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. Please fix the issue mentioned below and then request for code review

## Reason

- [x] Your JS and CSS linters are failing. Please fix that.

### Please make sure to submit a code review when you are done with the changes that was requested

**Don't worry you still have the same number of requests left as previous.**


Cheers and Happy Coding :clap: :clap: :clap: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

**_Dipesh Kumar_**

**Comment by kimothokamau** [comment]
I have resolved the linter errors

File: README.md [hunk 1]
```diff
@@ -1,70 +1,62 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `yarn build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+This project is [MIT](lic.url) licensed.
```
Comment (MahmoudBakr23) [inline README.md]: - [ ] Kindly fix the broken MIT link for a completely professional README file 😄 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @kimothokamau 😃, 

You did a great job setting up the application. However, there is just one small thing left to do in order to move on 💪 

### **Changes Required**

_Please see the comment down below._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @kimothokamau , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional
- MIT license is taking us not on the right page fix it or remove it completely


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**